### DISCUSSION TITLE
Is it possible to use git tags as the template?

### DISCUSSION THREAD

**Discussion by MartinThoma** [release-drafter/release-drafter#1134] [Q&A]
I'm the maintainer of PyPDF2 and I would like to simplify my life with release-drafter.

I already automatically create git tags which could pretty much be used as the Github release ([example](https://github.com/py-pdf/PyPDF2/tags)). I could change the message to actually contain markdown.

Is it possible to draft a release directly from a new tag (and only new tags - there are several commits on `main` which will not become a release; I don't want to have too many open drafts)

**Comment by jetersen** [comment]
See this workflow https://github.com/release-drafter/release-drafter/blob/master/.github/workflows/release.yml
